my-sms/Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'twitter-bootstrap-rails'
 group :development do
   gem 'rubocop'
   gem 'spring-commands-rspec'
+  gem 'faker'
```

Comment:
@josh-cooper looks like this should be in the next group .. ie to include :test as well as :development

Reply:
My preference was to only use it for development, for demo data. I was keeping a separate factory for tests using sequences as described in the Everyday Rails Guide. That way the test data is more predictable, making the tests more deterministic.
Though I'd prefer to follow the standard approach if that is to use it for tests as well.

Reply:
On second thought this might cause an error in the testing environment if any calls to Faker are left outside of blocks. Its safer just to include it as a test dependency even if it isn't used for testing at the moment.

Fixed in f1bf9a133f1e8d60d6bbead534339c5d48b6f8d1